libuavcan/include/uavcan/node/generic_publisher.hpp

```diff
@@ -81,50 +77,43 @@ class GenericPublisherBase : Noncopyable
  * DataSpec - data type specification class
  * DataStruct - instantiable class
  */
-template <typename DataSpec, typename DataStruct>
-class UAVCAN_EXPORT GenericPublisher : public GenericPublisherBase
+template <typename DataSpec, typename DataStruct> class UAVCAN_EXPORT GenericPublisher : public GenericPublisherBase
 {
     struct ZeroTransferBuffer : public StaticTransferBufferImpl
     {
-        ZeroTransferBuffer() : StaticTransferBufferImpl(UAVCAN_NULLPTR, 0) { }
+        ZeroTransferBuffer()
+            : StaticTransferBufferImpl(UAVCAN_NULLPTR, 0)
+        {
+        }
     };
 
-    typedef typename Select<DataStruct::MaxBitLen == 0,
-                            ZeroTransferBuffer,
-                            StaticTransferBuffer<BitLenToByteLen<DataStruct::MaxBitLen>::Result> >::Result Buffer;
-
-    enum
-    {
-        Qos = (DataTypeKind(DataSpec::DataTypeKind) == DataTypeKindMessage) ?
-              CanTxQueue::Volatile : CanTxQueue::Persistent
-    };
+    typedef typename Select<DataStruct::MaxBitLen == 0, ZeroTransferBuffer,
+                            StaticTransferBuffer<BitLenToByteLen<DataStruct::MaxBitLen>::Result>>::Result Buffer;
 
     int checkInit();
 
     int doEncode(const DataStruct& message, ITransferBuffer& buffer) const;
 
-    int genericPublish(const DataStruct& message, TransferType transfer_type, NodeID dst_node_id,
-                       TransferID* tid, MonotonicTime blocking_deadline);
+    int genericPublish(const DataStruct& message, TransferType transfer_type, NodeID dst_node_id, TransferID* tid,
+                       MonotonicTime blocking_deadline);
 
-public:
+  public:
     /**
      * @param max_transfer_interval     Maximum expected time interval between subsequent publications. Leave default.
      */
     GenericPublisher(INode& node, MonotonicDuration tx_timeout,
                      MonotonicDuration max_transfer_interval = TransferSender::getDefaultMaxTransferInterval())
         : GenericPublisherBase(node, tx_timeout, max_transfer_interval)
-    { }
+    {
+    }
 
-    ~GenericPublisher() { }
+    ~GenericPublisher() {}
 
     /**
      * Init method can be called prior first publication, but it's not necessary
      * because the publisher can be automatically initialized ad-hoc.
      */
-    int init()
-    {
-        return checkInit();
-    }
+    int init() { return checkInit(); }
```

Comment:
I wonder if there is any way to ask the auto-formatter to stop transforming functions into one-linters? This decision should be left to the developer, probably.